tests/test_catalogMatch.py

```diff
@@ -31,32 +31,7 @@
 from lsst.daf.base import PropertyList
 from lsst.daf.butler import DatasetRef, DatasetType, DimensionUniverse, StorageClass
 from lsst.meas.algorithms import ReferenceObjectLoader
-
-
-class MockSourceTableRef:
-    """Replicate functionality of `DeferredDatasetHandle`"""
-
-    def __init__(self, source_table, ref=None, dataId=None):
-        self.source_table = source_table
-        self.ref = ref
-        self.dataId = dataId
-
-    def get(self, parameters={}, **kwargs):
-        """Retrieve the specified dataset using the API of the Gen3 Butler.
-        Parameters
-        ----------
-        parameters : `dict`, optional
-            Parameter dictionary.  Supported key is ``columns``.
-        Returns
-        -------
-        dataframe : `pandas.DataFrame`
-            dataframe, cut to the specified columns.
-        """
-        if "columns" in parameters:
```

Comment:
It doesn't look like this parameter is ever used because it looks like this code has been copied from an implementation that had been used for pandas DataFrame objects -- the tests pass and SimpleCatalog is not configured to understand a columns parameter.